data/challenge missions.txt

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) 2014 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+
+
+mission "bronzeman bet"
+	landing
+	name "Collect on your bet."
+	description "You should pay your old 'friend' on New Boston a visit and collect the money he owes you."
+	destination "New Boston"
+	to offer
+		challengeMode == 1
+		not saveFileTampered
+		or
+			year > 3014
+			and
+				year = 3014
+				month > 2
+			and
+				year = 3014
+				month = 2
+				day >= 17

```

Comment:
I could have used an event to encode this date comparison, but it seemed like events were for more important things...